### PR TITLE
WOfS Alchemist YAML for USGS Collection 2 Landsat

### PR DESCRIPTION
**Pull request by eloise-b**
Adding the yaml for WOfS using USGS collection 2 data. 
This works on the sandbox using `run-one` and `run-many` in alchemist.

### CODE REVIEW

File: dev/alchemist/DEAfrica_wofs_c2.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+specification:
+  product: ls8_sr
+  measurements: ['blue', 'green', 'red', 'nir', 'swir_1', 'swir_2', 'pixel_quality']
+  #measurements: ['nbart_blue', 'nbart_green', 'nbart_red', 'nbart_nir', 'nbart_swir_1', 'nbart_swir_2', 'oa_fmask']
```
Comment (alexgleith) [inline dev/alchemist/DEAfrica_wofs_c2.yaml:4]: Let's remove this :-)

File: dev/alchemist/DEAfrica_wofs_c2.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  aws_unsigned: False
+  transform: wofs.virtualproduct.WOfSClassifier
+  transform_url: 'https://github.com/GeoscienceAustralia/wofs/'
+  override_product_family: ard # The relationship of the source dataset to the newly derived dataset
+  basis: green
+
+  transform_args:
+    c2_scaling: True
+    dsm_path:  's3://deafrica-input-datasets/srtm_dem/srtm_africa.tif'
+    #dsm_path:  'https://deafrica-data.s3-us-west-2.amazonaws.com/ancillary/dem/srtm_africa.tif' #'s3://deafrica-data/ancillary/dem/srtm_africa.tif'
```
Comment (alexgleith) [inline dev/alchemist/DEAfrica_wofs_c2.yaml:23]: Let's remove this too please

File: dev/alchemist/DEAfrica_wofs_c2.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+output:
+  location: '/home/jovyan/Alchemist/' #'s3://deafrica-data-dev/wofs/new'
```
Comment (alexgleith) [inline dev/alchemist/DEAfrica_wofs_c2.yaml:26]:
```suggestion
  location: s3://deafrica-data-dev-af/
```

File: dev/alchemist/DEAfrica_wofs_c2.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  nodata: 1
+  dtype: uint8
+  explorer_url: https://explorer.dev.digitalearth.africa/products/ls8_c2l2/extents
```
Comment (alexgleith) [inline dev/alchemist/DEAfrica_wofs_c2.yaml:29]: Can we make this the right name, whatever product it is that is the result...

File: dev/alchemist/DEAfrica_wofs_c2.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    naming_conventions: default
+
+processing:
+  dask_chunks:
+    x: 600
+    y: 600
+#  dask_client:
+#    processes: True
+#    threads_per_worker: 1
+#    n_workers: 4
```
Comment (alexgleith) [inline dev/alchemist/DEAfrica_wofs_c2.yaml:64]: Remove please

File: dev/alchemist/DEAfrica_wofs_c2.yaml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    dataset_version: 0.0.1
+  properties:
+    dea:dataset_maturity: interim
+    collection_number: 2
+    naming_conventions: default
+
+processing:
+  dask_chunks:
+    x: 600
+    y: 600
```
Comment (alexgleith) [inline dev/alchemist/DEAfrica_wofs_c2.yaml:60]: Let's change these, like this: https://github.com/GeoscienceAustralia/dea-config/blob/master/dev/services/alchemist/ga_ls_wo_3/ga_ls_wo_3.alchemist.yaml

**Review by alexgleith — CHANGES_REQUESTED** [review]
Can we please name this like the final product, so <product_name>.alchemist.yaml.

Other comments throughout.

**Comment by fangfy** [comment]
@eloise-b forgot to mention that I've also made a few changes in the version you made in the scripts repo. Have a look there and we should also clean that repo up I think.

**Comment by alexgleith** [comment]
We should probably clean this repo up a bit too! Todo: ...

**Review by eloise-b — COMMENTED** [review]
These changes have been completed.

**Pull request merged**